recipes/ribocode/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: ribocode
+  version: 1.2.6
+
+source:
+  fn: RiboCode-1.2.6.tar.gz
+  url: https://pypi.python.org/packages/c6/4b/902412b7c9683bcb9096e9359d17c6c0538b084a0c3167603ed69278aeb6/RiboCode-1.2.6.tar.gz
+  md5: e05ef843da351c4588e02bad3d5c3d65
+
+requirements:
+  build:
+    - python # [py27]
+    - numpy 
+    - pysam >0.8.4
+    - matplotlib
+    - scipy
+    - pyfasta
+    - biopython
+    - h5py
+    - htseq
+  run:
+    - python # [py27]
+    - numpy 
+    - pysam >0.8.4
+    - matplotlib
+    - scipy
+    - pyfasta
+    - biopython
+    - h5py
+    - htseq
+test:
+  imports:
+    - RiboCode
+  commands:
+    - RiboCode --help
+    - prepare_transcripts --help
+    - metaplots --help
+    - plot_orf_density --help
+    - ORF_count --help
+
+build:
+  skip: True # [not py27]
+  entry_points:
+    - RiboCode=RiboCode.RiboCode:main
+    - prepare_transcripts=RiboCode.prepare_transcripts:main
+    - metaplots=RiboCode.metaplots:main
+    - plot_orf_density=RiboCode.plot_orf_density:main
+    - ORFcount=RiboCode.RPF_count_ORF:main
+  script:
+    python setup.py install
```

Comment:
as you have this line here included you can remove this build.sh file completely ...